doc/conf_ref.rst

```diff
@@ -75,7 +75,8 @@ or :ref:`mirrorlist <mirrorlist-label>` option definition.
     When upgrading a package, always try to install its highest version
     available, even only to find out some of its deps are not
     satisfiable. Enable this if you want to experience broken dependencies in
-    the repositories firsthand. The default is False.
+    the repositories firsthand. The default is ``True``. The default is overwritten by configuration
+    in ``dnf.conf`` file (``/etc/dnf/dnf.conf``) that could differ for particular distribution.
```

Comment:
So this is the documentation of the configuration file, so the sentence:
>  The default is overwritten by configuration in ``dnf.conf`` file (``/etc/dnf/dnf.conf``)

Is redundant here, right? So I'd leave it with:
> The default is ``True``.

I think I'd leave out the mention of the distribution here too. Your wording kind of says the value can be different across various distributions, but the man page should be specific to the distribution of the user. That distribution has a very concrete value in the config, but we can't know it here. But the information is perhaps not that relevant to the user and they may have changed the value themself anyway.

But, if you wanna keep it in, maybe something like:
> The default is ``True``. This value may be set in your configuration file according to your distribution policy.